proposals/simd/BinarySIMD.md

```diff
@@ -132,12 +132,11 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
 | `i32x4.sub`                |    `0x7c`| -                  |
 | `i32x4.mul`                |    `0x7f`| -                  |
 | `i64x2.neg`                |    `0x84`| -                  |
-| `i64x2.any_true`           |    `0x85`| -                  |
-| `i64x2.all_true`           |    `0x86`| -                  |
 | `i64x2.shl`                |    `0x87`| -                  |
 | `i64x2.shr_s`              |    `0x88`| -                  |
 | `i64x2.shr_u`              |    `0x89`| -                  |
 | `i64x2.add`                |    `0x8a`| -                  |
+| `i64x2.mul`                |    `0x8c`| -                  |
```

Comment:
I thought we were reserving `0x90` for `i64x2.mul`.
`i8x16.mul`(`0x5d`) = `i8x16.neg`(`0x51`) + 12
`i16x8.mul`(`0x6e`) = `i16x8.neg`(`0x62`) + 12
`i32x4.mul`(`0x7f`) = `i32x4.neg`(`0x73`) + 12
`i64x2.mul`(`0x8c`) = `i64x2.neg`(`0x84`) + 8?

Reply:
Good point, I don't think these are final yet, we will look at the actual opcodes again once we have a complete set. There will probably be reordering required.

Reply:
We were pretty intentional in laying out the opcode space about a year ago before we started adding new instructions, so I think we should respect those decisions where possible.

Reply:
I'm happy to change the opcode of i64x2.ul, but with new instructions like the load_splat and load_extend, the space of i8x16 and the space of i64x2 will not be the same size anymore, and this nice equation mapping *.neg to *.mul will conflict with the goal of keeping all the i8x16 instructions together (supposing there is such a goal).

Reply:
https://github.com/WebAssembly/simd/pull/116

Reply:
Right, I agree that for new instructions like load_splat, load_extend, narrowing, and widening just putting them at the end of the opcode space for now is the best move.